contracts/entrypoint.sh

```diff
@@ -140,5 +137,9 @@ elif [ "${DEPLOY_TYPE}" = "validator-registry" ]; then
         -vvvv \
         --use 0.8.23 \
         --root "${CONTRACT_REPO_ROOT_PATH}" \
-        --via-ir
+        --via-ir \
+        --skip-simulation \
+        --force \
```

Comment:
Weird that they use force to describe clearing a cache, but good idea to keep it here!